pkg/database/logger.go

```diff
@@ -0,0 +1,134 @@
+package database
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+)
+
+const (
+	maximumCallerDepth int = 15
```

Comment:
maybe we can use this as a configuration in `LoggerAdaptorConfig` ?

Reply:
We could, but I don't see any reason to change this in the future. This is just to avoid looking at too many frames, but we know that the call stack within gorm won't be that big.